### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Import- Activity Update]: Keyboard focus is incorrect inside the Calendar window under 'Activity Update' pane.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8465]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1

Repro Steps:
1. Launch the Adaptive cards app on Android.
2.  Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Activity Update'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.
8. Navigate to Date Picker and activate it.
9. Observe the keyboard focus inside the Calendar window.

Actual Result:
Keyboard focus is incorrect inside the Calendar window under 'Activity Update' pane. Order: Selected date-> Date appearing above-> Cancel button-> OK button.

Expected Result:
Keyboard focus order should be in correct manner so that user can navigate without any hurdle. Keyboard focus should be logical and sequential inside the Calendar pop up.

User Experience:
Keyboard users will not be able to navigate the control efficiently if keyboard focus order is incorrect inside the Calendar window under 'Activity Update' pane.

WCAG Reference:
(https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7d)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/89897257/232707193-ab1cbf21-c64b-424c-ac88-7e5ab8f3b8ba.mp4

### Card JSON

```JSON
NA
```
